src/dfcx_scrapi/core/environments.py

```diff
@@ -32,13 +31,13 @@
 )
 
 
-class Environments(ScrapiBase):
+class Environments(scrapi_base.ScrapiBase):
     """Core Class for CX Environments Resource functions."""
 
     def __init__(
         self,
         creds_path: str = None,
-        creds_dict: dict = None,
+        creds_dict: Dict[str,str] = None,
         creds=None,
```

Comment:
@kmaphoenix type hint for creds for consistency

Reply:
ack.